README.md

```diff
@@ -106,6 +106,7 @@ scoop reset deno
 - The Shell installer can be used on Windows via the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about).
 
 ## Known Issues
+- The program `unzip` is a requirement for the deno install process
```

Comment:
This is only true for the Shell installer, not for the PowerShell installer. Can you please adjust the wording accordingly?